xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -2708,6 +2708,15 @@ bool CActiveAE::SupportsRaw(AEAudioFormat &format)
   return true;
 }
 
+bool CActiveAE::SupportsCoreExtraction(int profile)
+{
+  if (profile != FF_PROFILE_DTS_HD_HRA && profile != FF_PROFILE_DTS_HD_MA)
```

Comment:
I'm not sure whether this is correct or I should just test against all currently known non-HD DTS-subtypes.

Reply:
It's not that critical at the moment. We basically only support DTS core extraction. Therefore you can call this method with the AEAudioFormat itself, e.g.

```
  if ((format.m_streamInfo.m_type == CAEStreamInfo::STREAM_TYPE_DTSHD || format.m_streamInfo.m_type == CAEStreamInfo::STREAM_TYPE_DTSHD_MA) && (!m_settings.dtspassthrough || !m_settings.allowdtscorepassthrough))
    return false;

return true;
```